pkg/reconcilermanager/controllers/util.go

```diff
@@ -93,11 +93,7 @@ func reconcilerEnvs(clusterName, syncName, reconcilerName string, reconcilerScop
 	case v1beta1.HelmSource:
 		syncRepo = helmConfig.Repo
 		syncDir = helmConfig.Chart
-		if helmConfig.Version != "" {
-			syncRevision = helmConfig.Version
-		} else {
-			syncRevision = "latest"
```

Comment:
If the version is left out entirely, helm CLI will automatically pull the latest one, so there is no need to set a default here. It was discussed offline that supporting polling from a latest tag with helm could be inefficient/not scalable.

Reply:
We could also consider keeping it, so that if helm CLI decides to break this default behavior later, we wouldn't be affected.

Reply:
Thank you for your comment, as it led me to further investigate this code. I learned that this does not actually affect what gets passed to helm CLI as it populates the `reconciler` container, not the `helm-sync` container. It seems that the primary purpose of this default value is to display `version: latest` in the RSync status when the version is empty (rather than displaying `version: ""`). Since another part of my PR overwrites the RSync status to display the version that was actually pulled down, I think in most cases this bit of code is inconsequential. That said, if something were to go wrong with the version parsing, it would make more sense to display "latest" than "", so I have reverted this change.